packages/genspio/genspio.0.0.0/opam

```diff
@@ -0,0 +1,23 @@
+opam-version: "1.2"
+maintainer: "Seb Mondet <seb@mondet.org>"
+authors: "Seb Mondet <seb@mondet.org>"
+homepage: "http://www.hammerlab.org/docs/genspio/master/index.html"
+bug-reports: "https://github.com/hammerlab/genspio/issues"
+license: "Apache 2.0"
+dev-repo: "https://github.com/hammerlab/genspio.git"
+build: [
+  [make "byte"]
+  [make "native"]
+]
+install: [
+  [make "META"]
+  [make "genspio.install"]
```

Comment:
I think these can be part of `build`, as they just generate an `.install` file. No need for an `install` field